docs/components/date-picker.njk

```diff
@@ -0,0 +1,81 @@
+{% extends "component.njk" %}
```

Comment:
I think its worth in the docs mentioning the need to include the javascript library along with the component